src/xcode/ENA/ENA/Source/Models/Home/HomeRiskCellConfigurator/HomeRiskCellConfigurators/HomeInactiveRiskCellConfigurator.swift

```diff
@@ -59,48 +59,70 @@ final class HomeInactiveRiskCellConfigurator: HomeRiskCellConfigurator {
 		self.lastUpdateDate = lastUpdateDate
 	}
 
-	// MARK: Configuration
+	// MARK: - Computed properties.
```

Comment:
good idea to add these as computed properties. makes the configuration a lot cleaner